src/current/_includes/releases/v23.2/v23.2.0-alpha.4.md

```diff
@@ -0,0 +1,197 @@
+## v23.2.0-alpha.4
+
+Release Date: October 23, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-alpha-4-general-changes">General changes</h3>
+
+- Updated the `licenses/CCT.txt` file to reflect the latest [Cockroachdb Community License](https://www.cockroachlabs.com/cockroachdb-community-license/). [#112494][#112494]
+
+<h3 id="v23-2-0-alpha-4-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Renamed [cluster settings](../v23.2/cluster-settings.html) related to physical cluster replication for consistency. For example, `bulkio.stream_ingestion.minimum_flush_interval` is now `physical_replication.consumer.minimum_flush_interval`. [#111197][#111197]
+
+<h3 id="v23-2-0-alpha-4-sql-language-changes">SQL language changes</h3>
+
+- [`SHOW SCHEDULES`](../v23.2/show-schedules.html) has two columns that surface the schedule options. These columns have been renamed to align with the documented option names: `on_previous_running` and `on_execution_failure`. [#111759][#111759]
+- Added support for the PLpgSQL `CLOSE` statement, which allows a PLpgSQL routine to close a cursor with the name specified by a cursor variable. [#111330][#111330]
+- When a [`RESTORE`](../v23.2/restore.html) with `remove_regions` is performed, the restore job will now fail if the object contains a [`REGIONAL BY ROW`](../v23.2/multiregion-overview.html#table-locality) table. [#111443][#111443]
+- It is now possible to open a [cursor](../v23.2/cursors.html) within a PLpgSQL function or procedure with an exception block. If an error occurs, creation of the cursor is rolled back before control reaches the exception handler. [#111735][#111735]
+- If a [scheduled backup](../v23.2/create-schedule-for-backup.html) resumes on a new cluster (e.g., after physical cluster replication cutover or a cluster restore), the backup schedule will pause. The user may [resume the schedule](../v23.2/resume-schedules.html) without changing it, but should take special care to ensure no other schedule is backing up to the same [collection](../v23.2/take-full-and-incremental-backups.html#backup-collections). The user may also want to cancel the paused schedule and start a new one. [#111578][#111578]
+- Added support for PLpgSQL `FETCH` and `MOVE` statements. Similar to SQL `FETCH`/`MOVE` statements, commands that would seek the [cursor](../v23.2/cursors.html) backward will fail. In addition, expressions other than constant integers are not yet supported for the `count` option. [#111318][#111318]
```

Comment:
link "FETCH" to https://www.postgresql.org/docs/current/plpgsql-cursors.html#PLPGSQL-CURSOR-USING-FETCH
link "MOVE" to  https://www.postgresql.org/docs/current/plpgsql-cursors.html#PLPGSQL-CURSOR-USING-MOVE